frontend/src/views/ProducerFormPage/SubstancesTable.vue

```diff
@@ -31,13 +31,15 @@
 
         <!-- Cells des inputs (communes à toutes les substances) -->
         <div class="sm:table-cell ca-cell">
-          <div class="sm:hidden ca-xs-title">Quantité par DJR (en mg)</div>
+          <div class="sm:hidden ca-xs-title">Quantité par DJR (en {{ payload.substances[rowIndex].quantity }})</div>
```

Comment:
Je pense qu'ici on veut la substance.unit ?
```suggestion
          <div class="sm:hidden ca-xs-title">Quantité par DJR (en {{ payload.substances[rowIndex].substance.unit }})</div>
```

![image](https://github.com/betagouv/complements-alimentaires/assets/1225929/cfb2f305-d7e5-4174-93e3-85cc7f0c5b35)

Reply:
ici aussi 49fbf5356b2dcfe5d4ee57eae1891085a505dd05